server/statistics/hot_cache.go

```diff
@@ -17,11 +17,17 @@ import (
 	"context"
 
 	"github.com/tikv/pd/server/core"
+	"go.uber.org/atomic"
```

Comment:
Why not just using `sync/atomic` pkg here?

Reply:
`sync/atomic` also lgtm.